sophiread/FastSophiread/src/disk_io.cpp

```diff
@@ -87,13 +87,6 @@ mapinfo_t readTPX3RawToMapInfo(const std::string &tpx3file) {
   return info;
 }
 
-// for mmap support
-#include <errno.h>
```

Comment:
Generally, you leave the `#include`s in the header as small as possible and leave them in the cpp files. The code, in principle, will compiler faster. You don't have to change this.